pkg/aws/ec2/ec2.go

```diff
@@ -203,6 +206,18 @@ func parseENI(iface *ec2_types.NetworkInterface, vpcs ipamTypes.VirtualNetworkMa
 		}
 	}
 
+	for _, prefix := range iface.Ipv4Prefixes {
+		prefixCidr := prefix.Ipv4Prefix
```

Comment:
nit (see also comment below)
```suggestion
		prefixCidr := aws.ToString(prefix.Ipv4Prefix)
```

Allows you to get rid of the pointer de-referencing